xyz.openbmc_project.SessionManager.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=OpenBMC Session Manager service.
+
+After=dropbear.socket
+After=bmcweb.service
```

Comment:
It seems to me these conditions should be rather opposite or even totally removed.
What are they for?

Reply:
As far as I can see in the systemd-doc the `WantedBy=` used to specify an install path to resolve dependencies. The `After=` uses to set the order to start services.

Reply:
As I understood `bmcweb` and `dropbear` will now notify `sensor-manager` about new sessions. So, it seems to me `session-manager` should be started before them but not after.

Reply:
Exactly! The 'After=' should be replaced with 'Before='. Thanks, fixed.